website/siteConfig.js

```diff
@@ -77,6 +77,11 @@ const siteConfig = {
   // Open Graph and Twitter card images.
   ogImage: '',
   twitterImage: '',
+  algolia: {
+    apiKey: process.env.ALGOLIA_API_KEY,
+    indexName: 'TBD',
```

Comment:
what is the use of this `indexName`? When we need to change it?

Reply:
This is the name of the Algolia index where your records will be pushed. We don't determine this; they send us this name with the key.

Reply:
Here a little more of the config file
https://community.algolia.com/docsearch/config-file.html#index_name
And the `Docusaurus` setup docs
https://docusaurus.io/docs/en/search